components/cards/SessionCard.tsx

```diff
@@ -0,0 +1,271 @@
+              {truncate(50, item.title)}
+            </StyledText>
+          </View>
+
+          <Space size={12} />
+
+          <StyledText size="sm" font="light">
+            {getSessionTimeAndLocation(item.slug)}
+          </StyledText>
+        </View>
+      </View>
+    </TouchableWithoutFeedback>
+  );
+};
+
+const SessionCardOnSessions = (props: SessionCardProps<SessionForSchedule>) => {
+  const { handlePress, item, handleBookMark } = props;
+  const { colors } = useTheme();
+
+  return (
+    <TouchableWithoutFeedback testID="card-sessions" onPress={handlePress}>
+      <View style={[styles.card, styles.cardTertiary, { backgroundColor: colors.card }]}>
+        {item.session_image !== null ? (
+          <Image
+            source={{
+              uri:
+                item.session_image ||
+                'https://res.cloudinary.com/droidconke/image/upload/v1668083311/prod/upload/sessions/pwt9pnojtmng8bymxkls.png',
+            }}
+            style={styles.image}
+            contentFit="cover"
+            contentPosition="left"
+          />
+        ) : (
+          <Image
+            source={require('../../assets/images/banner.png')}
+            style={styles.image}
+            contentFit="cover"
+            contentPosition="center"
+          />
+        )}
+
+        <Space size={4} />
+
+        <View style={styles.bottomForSessions}>
+          <StyledText size="sm" font="light">
+            {getScheduleTimeAndLocation(item.start_date_time, item.end_date_time, item.rooms[0] as Room)}
+          </StyledText>
+          <Space size={11} />
+          <View>
+            <StyledText font="bold" numberOfLines={2} style={styles.title}>
+              {truncate(50, item.title)}
+            </StyledText>
+          </View>
+          <Space size={16} />
+          <Row>
+            <Row style={styles.avatarRow}>
+              {item.speakers.map((speaker) => (
+                <View key={speaker.avatar}>
+                  <Image
+                    source={{ uri: speaker.avatar || '' }}
+                    style={[styles.avatar, { borderColor: colors.primary }]}
+                  />
+                </View>
+              ))}
+            </Row>
+            <AntDesign
+              testID="bookmark"
+              name={item.is_bookmarked ? 'star' : 'staro'}
+              size={21}
+              color={item.is_bookmarked ? colors.tertiary : colors.primary}
+              onPress={handleBookMark}
+            />
+          </Row>
+        </View>
+      </View>
+    </TouchableWithoutFeedback>
+  );
+};
+
+const SessionCardList = (props: SessionCardProps<SessionForSchedule>) => {
+  const { handlePress, item, handleBookMark } = props;
+  const { colors } = useTheme();
+
+  const timeSplitted: Array<string> = getSessionTime(item.start_time).split(' ');
+
+  return (
+    <TouchableWithoutFeedback testID="card-list" onPress={handlePress}>
```

Comment:
`TouchableWithoutFeedback` is not working for me. When I click on the card in the list view, it does not trigger the `onPress` event. Do you have the same issue?

Reply:
I am also experiencing the same issue, lemme look into it.

Reply:
Changed it from TouchableWithoutFeedback to TouchableOpacity.